src/V3EmitCMain.cpp

```diff
@@ -74,10 +74,6 @@ class EmitCMain final : EmitCBaseVisitor {
              + "{contextp.get()}};\n");
         puts("\n");
 
-        puts("// Evaluate initials\n");
```

Comment:
The original reason for this is some signals were set between here and main loop.  Without that I think it is ok to remove this.